queue/src/fiskaltrust.Middleware.Queue/Bootstrapper/QueueBootstrapper.cs

```diff
@@ -57,21 +58,14 @@ public void ConfigureServices(IServiceCollection services)
 
         private static async Task CreateConfigurationActionJournalAsync(MiddlewareConfiguration middlewareConfiguration, IActionJournalRepository actionJournalRepository)
         {
-            var configurationCopy = new Dictionary<string, object>(middlewareConfiguration.Configuration)
+            var configuration = new Dictionary<string, object>
             {
                 { "MachineName", Environment.MachineName },
                 { "ProcessArchitecture", RuntimeInformation.ProcessArchitecture.ToString() },
                 { "OSArchitecture", RuntimeInformation.OSArchitecture.ToString() },
-                { "OSDescription", RuntimeInformation.OSDescription.ToString() }
+                { "OSDescription", RuntimeInformation.OSDescription.ToString() },
+                { "...", "redacted"}
```

Comment:
Just to confirm that we have an issue already for improving that. I would also remove the redacted field.